### ISSUE TITLE
Problems with live reload

### ISSUE THREAD

**Issue by filipjohansson** [petecoop/generator-express#227]
I just scaffolded a new project with the latest generator. But I could not get the live reload to work. I looked at an old project where the live reload was working and found the following changes in the `gulpfile.js`file.

This
```
}).on('readable', function () {
  this.stdout.on('data', function (chunk) {
    if(/^Express server listening on port/.test(chunk)){
      livereload.changed(__dirname);
  }
});
```
had changed to this
```
}).on('readable', () => {
  this.stdout.on('data', (chunk) => {
    if (/^Express server listening on port/.test(chunk)) {
      livereload.changed(__dirname);
    }
});
```
When I changed it back it started working as expected.

I also think I tracked it down to this commit: https://github.com/petecoop/generator-express/commit/0024aed060deade2d259f0082926d5386bc2646e

**Comment by petecoop** [comment]
thanks for catching this, it's due to the change in `this`. I will revert this

**Comment by petecoop** [comment]
Fixed and published as 2.16.1
